app-text/atril/atril-1.28.1.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE_LA_PUNT="yes"
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit mate python-any-r1 virtualx
+
+DESCRIPTION="Atril document viewer for MATE"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+
+LICENSE="FDL-1.1+ GPL-2+ GPL-3+ LGPL-2+ LGPL-2.1+"
+SLOT="0"
+
+IUSE="caja dbus debug djvu dvi epub +introspection keyring nls +postscript synctex t1lib test tiff xps"
+
+REQUIRED_USE="t1lib? ( dvi )"
+
+COMMON_DEPEND="
+	app-accessibility/at-spi2-core:2
+	app-text/poppler[cairo]
+	dev-libs/glib:2
+	dev-libs/libxml2:2
+	>=mate-base/mate-desktop-$(ver_cut 1-2)
+	sys-libs/zlib
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:3[introspection?]
+	x11-libs/libICE
+	x11-libs/libSM:0
+	x11-libs/libX11
+	x11-libs/cairo
+	x11-libs/pango
+	caja? ( mate-base/caja[introspection?] )
+	djvu? ( app-text/djvu:0 )
+	dvi? (
+		virtual/tex-base
+		t1lib? ( media-libs/t1lib:5 )
+	)
+	epub? (
+		<dev-libs/mathjax-3
+		net-libs/webkit-gtk:4.1
+	)
+	keyring? ( app-crypt/libsecret )
+	introspection? ( dev-libs/gobject-introspection:= )
+	postscript? ( app-text/libspectre )
+	synctex? ( virtual/tex-base )
+	tiff? ( media-libs/tiff:= )
+	xps? ( app-text/libgxps )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	gnome-base/gvfs
+	virtual/libintl
+"
+
+BDEPEND="${COMMON_DEPEND}
```

Comment:
Most likely not correct, which also makes `COMMON_DEPEND` useless as you only need `DEPEND` and `RDEPEND` above.